order-service/src/main/kotlin/cz/kss/proj/orderservice/model/order-contract.kt

```diff
@@ -0,0 +1,38 @@
+package cz.kss.proj.orderservice.model
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Table
+import java.math.BigDecimal
+import java.time.Instant
+
+
+@Table("k_order")
```

Comment:
nit: `order`

Reply:
I'm a little afraid that there will be a problem with that word is reserved